applications/omnipresence/src/omnip_subscriptions.erl

```diff
@@ -407,6 +407,7 @@ subscribe_to_record(JObj) ->
                         ,subscription_id=wh_json:get_value(<<"Subscription-ID">>, JObj)
                         ,proxy_route= wh_json:get_value(<<"Proxy-Route">>, JObj)
                         ,version=Version
+                        ,user_agent=wh_json:get_value(<<"User-Agent">>, JObj)
```

Comment:
In theory, `get_value/2` could return `<<>>` as well. `get_ne_binary/2` promises to only return `'undefined' | ne_binary()`